chat-sdk-core/src/main/java/co/chatsdk/core/interfaces/InterfaceAdapter.java

```diff
@@ -45,6 +46,14 @@
 
     Activity profileActivity (User user);
 
+    void addTab(Tab tab);
+    void addTab(int index, Tab tab);
+
+    void addTab(String title, int icon, Fragment fragment);
+    void addTab(int index, String title, int icon, Fragment fragment);
+
```

Comment:
Again, here this should be:
`void addTab(String title, int icon, Fragment fragment, int index);`